hw/ip/rv_core_ibex/dv/sva/rv_core_ibex_otp_sva_if.core

```diff
@@ -0,0 +1,20 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:rv_core_ibex_otp_sva_if:0.1"
+description: "Ibex to OTP Controller assertion interface."
+filesets:
+  files_dv:
+    depend:
+      - lowrisc:ip:rv_core_ibex_pkg
+      - lowrisc:ip:otp_ctrl_pkg
+      - lowrisc:prim:assert
+    files:
+      - rv_core_ibex_otp_sva_if.sv
```

Comment:
You also need to add an SVA bind file along with the interface, which binds the interface to `top_earlgrey` module, and connects the signals are the right hierarchy to the interface ports. 

Then, add the bind file to `sim_tops` variable in `chip_sim_cfg.hjson`.